reporting/impl/src/main/java/org/jboss/windup/reporting/export/ExportCSVFileRuleProvider.java

```diff
@@ -202,5 +170,34 @@ private String buildLinkString(Iterable<LinkModel> links)
             linksString.toString();
             return linksString.toString();
         }
+
+        private void writeCsvRecordForProject(String outputFolderPath, ProjectModel projectModel, String[] line)
+        {
+            if (!projectToFile.containsKey(projectModel.getName()))
+            {
+                CSVWriter writer = initCSVWriter(outputFolderPath + PathUtil.cleanFileName(projectModel.getName()) + ".csv");
+                projectToFile.put(projectModel.getName(), writer);
+            }
+            projectToFile.get(projectModel.getName()).writeNext(line);
+
+        }
+
+        private CSVWriter initCSVWriter(String path)
+        {
+            try
+            {
+                CSVWriter writer = new CSVWriter(
+                            new FileWriter(path), ';');
+                String[] headerLine = new String[] { "Rule Id", "Problem type", "Title", "Description", "Links", "Application", "File Name",
+                            "File Path", "Line", "Story points" };
+                writer.writeNext(headerLine);
+                return writer;
+            }
+            catch (IOException e)
+            {
+                e.printStackTrace();
```

Comment:
Yes, we should throw an exception and also print to STDERR that we could not create the CSV. The user should be notified, then the exception should be rethrown. This will ensure that the error also appears in the rule-providers report as a "failed" status for ExportCSVFileRuleProvider.